### PR TITLE
[RDV-110] ohif-viewer-static project implementation

### PR DESCRIPTION
**Pull request by aaronadrid-isv**
implementation of separate `@ischemaview/ohif-viewer-static` to be built from `@ischemaview/ohif-viewer.`

NOTE: v1.0.0 build assets have been sourced from a working build that includes newer `@ischemaview/dicom-rapidai-extension` than what is currently referenced as a dependency `@ischemaview/dicom-rapidai-extension@0.0.1`.  Dependencies need to be sorted out so that `@ischemaview/dicom-rapidai-extension` can be built and versioned independently of this monorepo. In the future this repo should not have the `platform/viewer-static/dist` source controlled.

### CODE REVIEW

**Review by HemantPatel30 — APPROVED** [review]

File: platform/viewer-static/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@ischemaview/ohif-viewer-static",
```
Comment (marcelo-nagatomo) [inline platform/viewer-static/package.json:2]: can we call this `@ischemaview/rapid-dicom-viewer`?

**Review by marcelo-nagatomo — CHANGES_REQUESTED** [review]

File: platform/viewer-static/package.json (hunk 1 above)
Comment (aaronadrid-isv) [reply platform/viewer-static/package.json:2]: That is fine, but are we ok with keeping the non-static project name `@ischemaview/ohif-viewer`?

Comment (aaronadrid-isv) [reply platform/viewer-static/package.json]: Reverted `viewer` => `@ohif/viewer` for now, renamed `viewer-static` => `@ischemaview/rapid-dicom-viewer` as discussed

**Comment by aaronadrid-isv** [comment]
I also deleted and re-added the static assets to the dist folder, as I had inadvertently added some assets from another build I attempted to run.

**Review by marcelo-nagatomo — APPROVED** [review]

**Pull request merged**